apps/src/templates/instructions/codeReview/ReviewNavigator.jsx

```diff
@@ -85,11 +105,11 @@ class ReviewNavigator extends Component {
     ) : (
       <Button
         text={javalabMsg.returnToMyProject()}
-        color={Button.ButtonColor.white}
+        color={color || Button.ButtonColor.white}
```

Comment:
Do we need the color parameter here because we're currently using this component for both v1 and v2 and they have different colors?  Or does this button appear in different colors depending on the state in the v2 design?

Reply:
We need both color temporarily because we are using this component for both v1 and v2 and they have different colors